src/make_copies.js

```diff
@@ -65,10 +65,18 @@ module.exports = async function main(
                 /%RANGE_START_ISO/g,
                 config.inclusiveRangeStart.toISOString()
               )
+              .replace(
+                /%RANGE_START_DATE_ISO/g,
```

Comment:
These directives need to be added to the readme

Reply:
Added!